src/github3/checks.py

```diff
@@ -0,0 +1,279 @@
+# -*- coding: utf-8 -*-
+"""This module contains all the classes relating to Checks."""
+from __future__ import unicode_literals
+
+from json import dumps
+
+from . import apps
+from . import decorators
+from . import models
+
+
+class CheckSuite(models.GitHubCore):
+    """The :class:`CheckSuite <CheckSuite>` object.
+
+    .. versionadded:: 1.3.0
+
+    Please see GitHub's `CheckSuite Documentation`_ for more information.
+
+    .. attribute:: status
+
+        The status of the Check Suite
+
+    .. attribute:: conclusion
+
+        The highest priority check run conclusion. If it has not completed this
+        will be None
+
+    .. attribute:: head_sha
+
+        The sha of the commit at the head of the branch checked
+
+    .. attribute:: head_branch
+
+        The branch checked
```

Comment:
Am I interpreting this correctly as "The branch the check was run against"?

Reply:
Yes, I believe it's intended to represent the name of the branch the pull request / push came from. I'll work on some better wording. GitHub's wording isn't super clear.